src/index.jsx

```diff
@@ -3,14 +3,17 @@ import { render } from 'react-dom';
 import { createStore } from 'redux';
 import { Provider } from 'react-redux';
 // import App from './components/App';
-import StudentHome from './components/StudentHome';
-import reducer from './reducers/sampleReducer';
+import App from './components/App';
+import reducer from './reducers/AccountPageReducer';
```

Comment:
We're going to need to combine reducers when we have multiple files of reducers, but for now this is good :+1: